src/anm/animation/tween.js

```diff
@@ -254,6 +254,22 @@ Tween.register(C.T_ROTATE, function(values) {
     }
 });
 
+Tween.register(C.T_BONE_ROTATE, function(values) {
+    var _from = values[0],
+        to = values[1];
+    return function(t) {
+        this.angle = _from * (1.0 - t) + to * t;
+    }
+});
+
+Tween.register(C.T_BONE_LENGTH, function(values) {
+    var _from = values[0],
+        to = values[1];
+    return function(t) {
+        this.angle = _from * (1.0 - t) + to * t;
```

Comment:
You may safely do `+=` in these tweens. See [rotate-to-path tween](https://github.com/Animatron/player/blob/develop/src/anm/animation/tween.js#L263) implementation. But also please notice that tweens like this one should exist during the whole lifetime of an element: that's why [here](https://github.com/Animatron/player/blob/develop/src/import/animatron-importer.js#L302) we add a rotate tween with first value of an angle at start and a tween with final value of an angle in the end.